data/mods/Aftershock/mobs/zombies.json

```
@@ -518,5 +518,164 @@
       "FILTHY",
       "GUILT"
     ]
+  },
+  {
+    "id": "mon_moxie_elder",
+    "type": "MONSTER",
+    "name": { "str": "MoxPhore Elder" },
+    "description": "An arachnid humanoid stands here, Its claw-like limbs wrapped its body and head buried deep within its chest as it surveys the area around it.",
```

```suggestion
    "description": "An alien arachnid whose appearance barely betrais it apparently human origin.  Numerous claw-like limbs closely wrap and protect  its head and thorax.",
```
The phrase `Arachnid humanoid` doesnt sound that good. The general wording was a bit strange too.

---

"betrais it" -> "betrays its"